conf/messages.en

```diff
@@ -313,12 +313,11 @@ declaration.declarant-details.help.bodyText = Go to data element 3/18 Declarant
 declaration.declarant.eori.empty = Enter your EORI number
 declaration.declarant.eori.error.format = Enter a valid EORI number, for example GB123456789000
 
-supplementary.consignee.title = Who is the consignee?
+supplementary.consignee.title = What is the consignee’s address?
```

Comment:
As I remember this require double `'`. One `'` will be skipped.

Reply:
Thanks, fixed this now